src/java/com/puppetlabs/puppetserver/ShellUtils.java

```diff
@@ -66,9 +66,7 @@ private static ExecutionResult executeExecutor(CommandLine commandLine,
      */
     public static ExecutionResult executeCommand(String command)
             throws InterruptedException, IOException {
-        CommandLine commandLine = CommandLine.parse(command);
```

Comment:
This parse is needed for the jruby layer. It passes the command and arguments in as a single string. So rather than trying to parse them sanely in ruby, I chose to use the commons-exec facility to do so.